docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+nightscout:
+  build: .
+  links:
+   - database
+   - broker
+  ports:
+   - "1337:1337"
+  environment:
+   - MONGO_CONNECTION=mongodb://database/nightscout
+   - API_SECRET=mylittlesecret
```

Comment:
I don't think we should have a default api secret, would make it too easy for people to use it